src/Microsoft.AspNetCore.SignalR.Common/Internal/Protocol/InvocationMessage.cs

```diff
@@ -42,4 +42,11 @@ public override string ToString()
             return $"Invocation {{ {nameof(InvocationId)}: \"{InvocationId}\", {nameof(NonBlocking)}: {NonBlocking}, {nameof(Target)}: \"{Target}\", {nameof(Arguments)}: [ {string.Join(", ", Arguments.Select(a => a?.ToString()))} ] }}";
         }
     }
+
+    public class CancelInvocationMessage : HubMessage
```

Comment:
Put it in it's own file. I know it's small, but it'd be the only one that is in a different file if we didn't